salt/grains/core.py

```diff
@@ -1506,17 +1509,6 @@ def _linux_bin_exists(binary):
         return False
 
 
-def _get_interfaces():
-    '''
-    Provide a dict of the connected interfaces and their ip addresses
-    '''
-
-    global _INTERFACES
```

Comment:
The global `_INTERFACES` should be removed from this file: https://github.com/openSUSE/salt/blob/56701aebb0937251fb128b3e9a6ce914efb27e8f/salt/grains/core.py#L112

since we have it now in `salt.utils.network`:

https://github.com/openSUSE/salt/blob/56701aebb0937251fb128b3e9a6ce914efb27e8f/salt/utils/network.py#L58

Reply:
Hi Pablo I done your requested changes